common/src/main/java/me/shedaniel/architectury/event/events/CommandRegistrationEvent.java

```diff
@@ -27,9 +27,16 @@
 
 public interface CommandRegistrationEvent {
     /**
-     * Invoked after server registers its commands, equivalent to forge's {@code RegisterCommandsEvent} and fabric's {@code CommandRegistrationCallback}.
```

Comment:
This provides some detail that isn't on the method javadoc (the "invoked after" part), so I'd keep this here.